pkgs/applications/video/animdl/default.nix

```diff
@@ -0,0 +1,28 @@
+{ fetchFromGitHub, poetry2nix, lib }:
+poetry2nix.mkPoetryApplication rec {
+  pname = "animdl";
+  version = "1.7.17";
+  src = fetchFromGitHub {
+    owner = "justfoolingaround";
+    repo = "animdl";
+    rev = "fd3277fb919c4b5e7ef42daba0bd001695a9c124";
+    hash = "sha256-pgtrStorkIJJ3oIGgRjNWYwcQ5jUC8IS5iI9CyDJkzE=";
+  };
+  projectDir = src;
+  overrides = poetry2nix.defaultPoetryOverrides.extend (self: super: {
+    comtypes = super.comtypes.overridePythonAttrs
+      ( old: { buildInputs = (old.buildInputs or [ ]) ++ [ super.setuptools ]; });
+    anitopy = super.anitopy.overridePythonAttrs
+      ( old: { buildInputs = (old.buildInputs or [ ]) ++ [ super.setuptools ]; });
+    anchor-kr = super.anchor-kr.overridePythonAttrs
+      ( old: { buildInputs = (old.buildInputs or [ ]) ++ [ super.setuptools ]; });
+  });
```

Comment:
These packages need individual packaging.

Reply:
How would I do that?

Reply:
Visit https://nixos.org/manual/nixpkgs/unstable/ and search for

> Python library packages in Nixpkgs

Reply:
Done. Read the other comment I posted